### PR TITLE
Feature/fix headers entries iterator

### PR DESCRIPTION
**Pull request by vedrani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shoutem/redux-io",
-  "version": "2.2.0-beta.1",
+  "version": "2.2.0-beta.2",
```
Comment (dodsky) [inline package.json]: Should it be beta if we're merging to develop?

File: test/rsaa.spec.js [hunk 2]
```diff
@@ -78,5 +78,96 @@ describe('rsaa', () => {
...
+      ...headers,
+    };
+
+    const extractedHeaders = extractHeaders(resource);
+
+    expect(extractedHeaders).to.deep.equal(expectedHeaders);
+  });
+
+  it ('can extract headers in resource with empty headers', () => {
+    const headers = {};
```
Comment (dodsky) [inline test/rsaa.spec.js]: Is this for readability? I'd expect to see just `new Headers()`. Here and on L138.

File: test/rsaa.spec.js [hunk 3]
```diff
@@ -78,5 +78,96 @@ describe('rsaa', () => {
     expect(types[1].type).to.equal(LOAD_SUCCESS);
     expect(types[1].meta).to.not.be.undefined;
     expect(types[1].meta(...metaResponse)).to.deep.equal(expectedResponseMeta);
-  })
-})
+  });
+
+  it ('can extract headers from raw headers map', () => {
```
Comment (dodsky) [inline test/rsaa.spec.js]: should all these tests be in `describe('headers')` ?

File: package.json (hunk 1 above)
Comment (vedrani) [inline package.json]: Yes, we have git flow for v2, with master tagged for production releases

File: test/rsaa.spec.js (hunk 2 above)
Comment (vedrani) [inline test/rsaa.spec.js]: No, they are single source of all headers (input, expected, calculated)

**Pull request merged**